package.json

```diff
@@ -9,7 +9,9 @@
     "test:watch": "npm test -- --watch --growl",
     "start": "webpack-dev-server --config config/webpack.dev.config.js --hot --inline",
     "build": "webpack -p --config config/webpack.prod.config.js",
-    "lint": "eslint -c config/.eslintrc **/*.js"
+    "lint": "eslint -c config/.eslintrc **/*.js",
+    "precommit": "npm run all",
```

Comment:
i think running everything a commit basis is kinda overkill. =/ some people enjoys organizing patches into small commits (like me), so it would not be a good experience to have it running over and over again.